go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -66,21 +66,43 @@ func TestQueries(t *testing.T) {
 func TestSingleQuery(t *testing.T) {
 	t.Skip()
 
+	harness := newDoltHarness(t)
```

Comment:
in general it is courteous to limit unnecessary changes that make PRs more difficult to read